From 46ec9ab112339d7585d24209eb710adb0bdb87c3 Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Mon, 28 May 2018 11:03:56 +0800
Subject: [PATCH 3884/5242] MLK-18407: clk: accommodate scfw change for QXP PI
 ss

commit  7f266f91aace482e4332f902dae32d7f6aa6a258 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change pixel clock register of qxp PI ss in order to
accommodate scfw change for PI ss

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit 1a769a426f4dcbd145280b3ff613607fbf6bcaa4)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8qxp.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/clk/imx/clk-imx8qxp.c b/drivers/clk/imx/clk-imx8qxp.c
index 3ed7d9d..6e7e1f5 100644
--- a/drivers/clk/imx/clk-imx8qxp.c
+++ b/drivers/clk/imx/clk-imx8qxp.c
@@ -564,8 +564,8 @@ static int imx8qxp_clk_probe(struct platform_device *pdev)
 	clks[IMX8QXP_PARALLEL_CSI_CLK_SEL] = imx_clk_mux2_scu("pll0_sel", pll0_sels, ARRAY_SIZE(pll0_sels), SC_R_PI_0, SC_PM_CLK_PER);
 	clks[IMX8QXP_PARALLEL_CSI_PER_CLK_DIV] = imx_clk_divider2_scu("parallel_per_clk", "pll0_sel", SC_R_PI_0, SC_PM_CLK_PER);
 	clks[IMX8QXP_PARALLEL_CSI_MCLK_DIV] = imx_clk_divider_scu("parallel_csi_mclk_div", SC_R_PI_0, SC_PM_CLK_MISC0);
-	clks[IMX8QXP_PARALLEL_CSI_PIXEL_CLK] = imx_clk_gate2_scu("parallel_pixel_clk", "parallel_per_clk", (void __iomem *)(PARALLEL_CSI_LPCG + 0x18), 0, FUNCTION_NAME(PD_PARALLEL_CSI));
-	clks[IMX8QXP_PARALLEL_CSI_IPG_CLK] = imx_clk_gate2_scu("parallel_ipg_clk", "parallel_per_clk", (void __iomem *)(PARALLEL_CSI_LPCG + 0x4), 16, FUNCTION_NAME(PD_PARALLEL_CSI));
+	clks[IMX8QXP_PARALLEL_CSI_PIXEL_CLK] = imx_clk_gate_scu("parallel_pixel_clk", "parallel_per_clk", SC_R_PI_0, SC_PM_CLK_PER, (void __iomem *)(PARALLEL_CSI_LPCG + 0x18), 0, 0);
+	clks[IMX8QXP_PARALLEL_CSI_IPG_CLK] = imx_clk_gate_scu("parallel_ipg_clk", "parallel_per_clk", SC_R_PI_0, SC_PM_CLK_PER, (void __iomem *)(PARALLEL_CSI_LPCG + 0x4), 16, 0);
 	clks[IMX8QXP_PARALLEL_CSI_MISC0_CLK] = imx_clk_gate_scu("parallel_csi_mclk", "parallel_csi_mclk_div", SC_R_PI_0, SC_PM_CLK_MISC0, (void __iomem *)(PARALLEL_CSI_LPCG + 0x1C), 0, 0);
 
 
-- 
1.7.9.5

